src/WebJobs.Script/Scale/BlobStorageConcurrencyStatusRepository.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#nullable enable
+
+using System;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Storage.Blobs;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.WebJobs.Host.Executors;
+using Microsoft.Azure.WebJobs.Host.Scale;
+using Microsoft.Azure.WebJobs.Logging;
+using Microsoft.Azure.WebJobs.Script;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// TODO: TEMP - this temporary implementation should be removed once https://github.com/Azure/azure-webjobs-sdk/issues/2710 is addressed.
```

Comment:
This is the required storage v12 implementation of the SDK service [here](https://github.com/Azure/azure-webjobs-sdk/blob/33eb24a6a91facb5399dd59f50a363616826f4bf/src/Microsoft.Azure.WebJobs.Host.Storage/BlobStorageConcurrencyStatusRepository.cs#L21).